packages/flutter_tools/test/commands.shard/hermetic/generate_localizations_test.dart

```diff
@@ -18,17 +20,22 @@ import '../../src/test_flutter_command_runner.dart';
 
 void main() {
   FileSystem fileSystem;
+  BufferLogger logger;
+  Artifacts artifacts;
+  ProcessManager processManager;
 
   setUpAll(() {
     Cache.disableLocking();
   });
 
   setUp(() {
     fileSystem = MemoryFileSystem.test();
+    logger = BufferLogger.test();
+    artifacts = Artifacts.test();
+    processManager = const LocalProcessManager();
```

Comment:
Is this running real sub-processes? If so, it will need a real file system, and should be moved to the commands.shard/permable directory.

Reply:
I changed this so that we use a `FakeProcessManager` instead.